src/server/tests.rs

```diff
@@ -3135,3 +3135,25 @@ async fn compute_diagnostics_multi_file() {
 
     assert_eq!(0, diagnostics_again.len());
 }
+
+#[test]
+async fn compute_diagnostics_multi_file_error_in_another_file() {
+    let server = create_server();
+
+    let filename: String = "file:///path/to/script.flux".into();
+    let fluxscript = r#"from(bucket: "my-bucket")
+|> range(start: -100d)
+|> filter(fn: (r) => r.anTag == v.a)"#;
+    open_file(&server, fluxscript.into(), Some(&filename)).await;
+    open_file(
+        &server,
+        r#"v = a"#.to_string(),
+        Some("file:///path/to/an_vars.flux"),
+    )
```

Comment:
So, as I understand it we assume a default package name when nothing is specified, putting both of these scripts into the same package.

Should there be another test to verify the behavior across different packages? Maybe we're already covering this well enough elsewhere in the suite.

Reply:
Yeah, I think we're already covering it enough elsewhere in the suite. As summary, if _other_ files claim to belong to another package in the same directory, we currently just straight up ignore them. That's not great, but there are some flux changes that need to be made in order for an error ta actually occur properly. See https://github.com/influxdata/flux-lsp/blob/master/src/server/store.rs#L182